APMrover2/Rover.h

```diff
@@ -60,6 +60,7 @@
 #include <AP_ServoRelayEvents/AP_ServoRelayEvents.h>
 #include <AP_Stats/AP_Stats.h>                      // statistics library
 #include <AP_Terrain/AP_Terrain.h>
+#include <AP_RPM/AP_RPM.h>
```

Comment:
a small thing but I've actually put the includes in alphabetical order in Rover..

Reply:
fixed